project-base/templates/Front/Layout/base.html.twig

```diff
@@ -27,7 +25,8 @@
 
         {% block html_body %}{% endblock %}
 
-        {% block javascripts_bottom %}{% endblock %}
```

Comment:
removing this block means that developers do not have option, where page JS can be placed (head / bottom of page). Head javascript can have reason (eg. measuring scripts). I do not see reason for only one block in end of page. Two blocks (with better naming - head/bottom) make more sense to me.

Reply:
I reverted this change. Blocks are now `javascripts` and `javascripts_bottom` (as before).
As much I like better naming, I don't want to introduce this change to avoid unnecessary changes in the project during upgrading.